.env-docker

```diff
@@ -0,0 +1,4 @@
+MYSQL_ROOT_PASSWORD=mypass
+SECRET_KEY_BASE=abcdefg
+VIRTUAL_HOST=beer.docker
```

Comment:
But the `VIRTUAL_HOST` env variable is only used in the app containers. So this specific one is actually no redundant. The others are fine in here.